### ISSUE TITLE
Issue with multi-line strings in Rspec spec descriptions

### ISSUE THREAD

**Issue by gbp** [samphippen/rubyfmt#228]
* Ruby version: ruby 2.6.6p146 (2020-03-31 revision 67876) [x86_64-darwin19]
* Rubyfmt git sha: 3435618

## Input file

```ruby
RSpec.describe MyClass do
  it 'foo ' \
     'bar ' \
     'baz' do
  end
end
```

## Rubyfmt's output

```
RSpec.describe(MyClass) do
  it(
    "foo " \
      "bar " \

      "baz"  ) do
  end
end
```

## This breaks because

Output is syntactically incorrect, when running Rspec, Ruby errors with:
```
  ./spec/test_spec.rb:15: syntax error, unexpected tSTRING_BEG, expecting ')'
        "baz"  ) do
        ^
  ./spec/test_spec.rb:15: syntax error, unexpected ')', expecting end
        "baz"  ) do
               ^
```

**Comment by fables-tales** [comment]
I've confirmed this bug is still live in V0.6.7, output is:

```
RSpec.describe MyClass do
  it 'foo ' \
     'bar ' \
     'baz' do
  end
end
```

**Comment by reese** [comment]
This should be fixed in #343, the output on master is the following:

```ruby
RSpec.describe MyClass do
  it(
    "foo " \
      "bar " \
      "baz"
  ) do
  end
end
```
